backend/webserver/views.py

```diff
@@ -1,3 +1,51 @@
+import profile
+import re
 from django.shortcuts import render
+#from rest_framework import viewsets
+from rest_framework.views import APIView
+from .models import Author
+from .serializers import AuthorSerializer
+from rest_framework.response import Response
+
+class AuthorsView(APIView):
+    def get(self,request,*args, **kwargs):
+        
+        authors = Author.objects.all()
+        serializer = AuthorSerializer(authors,many=True)
+
+        return Response(serializer.data)
+    
+    def post(self,request,*args, **kwargs):
+        author_data = request.data
+
+        new_author = Author.objects.create(display_name=author_data["display_name"],profile_image=author_data["profile_image"],github_handle=author_data["github_handle"])
+
+        new_author.save()
```

Comment:
You shouldn't be saving the record to the database before validating the data given to us with the `POST` request. This following example (taken from https://blog.logrocket.com/django-rest-framework-create-api/#setting-up-django-rest-framework) shows how you can validate the data: 
![image](https://user-images.githubusercontent.com/43586048/196051041-408b72e4-f056-45de-8228-d399a49d0d2d.png)

If you don't validate the data beforehand, it can lead to faulty data in the database.